gnucash/gnome/assistant-hierarchy.cpp

```diff
@@ -119,13 +124,15 @@ typedef struct
     gboolean use_defaults;
     gboolean new_book;  /* presumably only used for new book creation but we check*/
 
-    GNCOptionDB *options;
+    GncOptionDB *options;
     GNCOptionWin *optionwin;
 
     GncHierarchyAssistantFinishedCallback when_completed;
 
 } hierarchy_data;
 
+extern gboolean gnc_book_options_dialog_apply_helper(GncOptionDB * options);
```

Comment:
Should this be in a common header rather than marking it extern in several files ?

Reply:
It's already declared in gnc-main-window.h, though it's also used only in assistant-hierarchy.cpp. But no point in declaring it twice, removed.